README.md

```diff
@@ -1,463 +1,475 @@
+The `canonicalize` parameter toggles whether the domain parameters get normalized with `canonicalDomain` or not.
 
-### `defaultPath(path)`
+#### `defaultPath(path)`
 
-Given a current request/response path, gives the Path apropriate for storing in a cookie.  This is basically the "directory" of a "file" in the path, but is specified by Section 5.1.4 of the RFC.
+Given a current request/response path, gives the path appropriate for storing in a cookie. This is basically the "directory" of a "file" in the path, but is specified by Section 5.1.4 of the RFC.
 
-The `path` parameter MUST be _only_ the pathname part of a URI (i.e. excludes the hostname, query, fragment, etc.).  This is the `.pathname` property of node's `uri.parse()` output.
+The `path` parameter MUST be _only_ the pathname part of a URI (excluding the hostname, query, fragment, and so on.). This is the `.pathname` property of node's `uri.parse()` output.
 
-### `pathMatch(reqPath,cookiePath)`
+#### `pathMatch(reqPath, cookiePath)`
 
-Answers "does the request-path path-match a given cookie-path?" as per RFC6265 Section 5.1.4.  Returns a boolean.
+Answers "does the request-path path-match a given cookie-path?" as per RFC 6265 Section 5.1.4. Returns a boolean.
 
 This is essentially a prefix-match where `cookiePath` is a prefix of `reqPath`.
 
-### `parse(cookieString[, options])`
+#### `parse(cookieString[, options])`
 
-alias for `Cookie.parse(cookieString[, options])`
+Alias for [`Cookie.parse(cookieString[, options])`](#cookieparsecookiestring-options).
 
-### `fromJSON(string)`
+#### `fromJSON(string)`
 
-alias for `Cookie.fromJSON(string)`
+Alias for [`Cookie.fromJSON(string)`](#cookiefromjsonstrorobj).
 
-### `getPublicSuffix(hostname)`
+#### `getPublicSuffix(hostname)`
 
-Returns the public suffix of this hostname.  The public suffix is the shortest domain-name upon which a cookie can be set.  Returns `null` if the hostname cannot have cookies set for it.
+Returns the public suffix of this hostname. The public suffix is the shortest domain name upon which a cookie can be set. Returns `null` if the hostname cannot have cookies set for it.
 
 For example: `www.example.com` and `www.subdomain.example.com` both have public suffix `example.com`.
 
-For further information, see http://publicsuffix.org/.  This module derives its list from that site. This call is currently a wrapper around [`psl`](https://www.npmjs.com/package/psl)'s [get() method](https://www.npmjs.com/package/psl#pslgetdomain).
+For further information, see the [Public Suffix List](http://publicsuffix.org/). This module derives its list from that site. This call is a wrapper around [`psl`](https://www.npmjs.com/package/psl)'s [`get` method](https://www.npmjs.com/package/psl##pslgetdomain).
 
-### `cookieCompare(a,b)`
+#### `cookieCompare(a, b)`
 
 For use with `.sort()`, sorts a list of cookies into the recommended order given in the RFC (Section 5.4 step 2). The sort algorithm is, in order of precedence:
```

Comment:
Include link to [Section 5.4](https://datatracker.ietf.org/doc/html/rfc6265#section-5.4) of the RFC